app/controllers/concerns/v1/profile_audit.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module V1
+  # Audit logging methods for Profiles
+  module ProfileAudit
+    extend ActiveSupport::Concern
+
+    private
+
+    def audit_creation
+      audit_success(
+        "Created policy #{new_policy.id} with initial profile" \
+        " #{new_profile.id} including host assinment and tailoring"
+      )
+    end
+
+    def audit_bo_creation(business_objective)
+      audit_success("Created Business Objective #{business_objective.id}")
+    end
+
+    def audit_tailoring_file
+      audit_success(
+        "Sent computed tailoring file #{tailoring_filename}" \
+        " for profile #{profile.id} of policy #{profile.policy_id}"
```

Comment:
These messages in controllers add more audit information. There is on message generated with the server response.l There is a `transaction_id` that ties them together. The last message also includes the controller. I might add file tracking to the payload (which I've experimented with)...

Proposed message format is welcomed. Keep in mind that the payload includes more information that the string message. These audit logs are supposed to be read by a human, an auditor in case of an event (or a request). They should provide as much necessary info to track what has happened.

The `after_action` is already handled by the MW. Controllers can potentially add more info to the context, keeping just one message per request.